tools/bug-report/pkg/bugreport/bugreport.go

```diff
@@ -61,8 +63,16 @@ func Cmd(logOpts *log.Options) *cobra.Command {
 		Use:          "bug-report",
 		Short:        "Cluster information and log capture support tool.",
 		SilenceUsage: true,
-		Long: "This command selectively captures cluster information and logs into an archive to help " +
-			"diagnose problems. It optionally uploads the archive to a GCS bucket.",
+		Long: "bug-report selectively captures cluster information and logs into an archive to help " +
+			"diagnose problems and optionally uploads the archive to a GCS bucket. \n" +
+			"Proxy logs can be filtered using:\n" +
+			"  --include|--exclude namespace1,namespace2.../deployment1,.../pod1,.../container1,.../label1=value1,.../annotation1=value1,...\n" +
+			"Exclude is applied after include." +
+			"All components are optional and can be omitted, which selects all.\n" +
+			"All names except label and annotation keys support '*' glob matching pattern.\n" +
+			"e.g. --include ns1,ns2 (only namespaces ns1 and ns2)\n" +
+			"     --include n*//p*/lbl=v* (pods with name beginning with 'p' in namespaces\n" +
```

Comment:
what are `/` used for? And what does `//` mean?

Reply:
From my understanding, the parameter passed to include is a comma separated list of paths of format `namespace/deployment/pod/label`. The slashes separate elements, and `//` matches anything in that category, so e.g. `naemspace//pod` will match anything like pod in namespace without filtering by deployment.

Reply:
@ostromart is that correct?

Reply:
yes, i updated the help text to make that a bit clearer.